GOVERNANCE.md

```diff
@@ -0,0 +1,53 @@
+# Jaeger Governance
+
+This document defines governance policies for the Jaeger project.
+
+## Maintainers
+
+Jaeger Maintainers have write access to the Jaeger GitHub repository https://github.com/jaegertracing/jaeger.
+They can merge their own patches or patches from others.
+
+This privilege is granted with some expectation of responsibility: maintainers are people who care about the Jaeger project and want to help it grow and improve. A maintainer is not just someone who can make changes, but someone who has demonstrated his or her ability to collaborate with the team, get the most knowledgeable people to review code, contribute high-quality code, and follow through to fix issues (in code or tests).
+
+A maintainer is a contributor to the Jaeger project's success and a citizen helping the project succeed.
+
+## Becoming a Maintainer
+
+To become a maintainer you need to demonstrate the following:
+
+  * commitment to the project
+    * participate in discussions, contributions, code reviews for 3 months or more,
+    * perform code reviews for 10 non-trivial pull requests,
+    * contribute 10 non-trivial pull requests and have them merged into master,
+  * ability to write good code,
+  * ability to collaborate with the team,
+  * understanding of how the team works (policies, processes for testing and code review, etc), 
+  * understanding of the projects' code base and coding style.
+
+A new maintainer must be proposed by an existing maintainer by sending an email to jaeger-maintainers@lists.cncf.io
```

Comment:
this should be done via a public forum, whether a public mailing list or github issue